src/http/api/resources/pubsub.js

```diff
@@ -20,10 +21,10 @@ exports.subscribe = {
 
     const handler = (msg) => {
       res.write(JSON.stringify({
-        from: msg.from,
+        from: bs58.decode(msg.from).toString('base64'),
```

Comment:
ah, missed this! This should continue as b58String. Will revert

Reply:
https://github.com/ipfs/js-ipfs-api/commit/470777d00eb94906d6aca57b2034814d49e081f0#commitcomment-25652929

Reply:
After all I didn't miss it, in js-ipfs-api it is correct. here is where it is wrong.

We want this value to be a b58String. The reason why the http-api sends base64 is just because how JSON serialization works in go-ipfs, the users should not have to worry about that.